### PR TITLE
chore: update ghas with warnings

### PR DESCRIPTION
**Pull request by notnmeyer**
i think this is the rest of the deprecation warnings 🤞 

closes https://github.com/opticdev/issues/issues/29

### CODE REVIEW

File: .github/workflows/bump.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
           title: Bumped to ${{ env.VERSION }} [ ${{ env.TYPE }} update]
           labels: ${{ env.TYPE }}
       - name: Comment back with PR
-        uses: unsplash/comment-on-pr@master
+        uses: unsplash/comment-on-pr@master # leaving this on master, the most recent tag is well behind
```
Comment (devdoshi) [inline .github/workflows/bump.yml:48]: ah that's annoying

**Review by devdoshi — APPROVED** [review]

**Comment by notnmeyer** [comment]
more changes coming, i still see some warning in some of the runs

**Comment by notnmeyer** [comment]
build failing with,

```
@useoptic/diff-engine
29
 | error Command "ws:build" not found.
```

```
➜ rg ws:build workspaces/diff-engine/
workspaces/diff-engine/package.json
7:    "ws:build-binaries": "scripts/build-binaries.sh",
```

**Comment by devdoshi** [comment]
Pushed a commit to fix the failing job - see commit message

**Comment by notnmeyer** [comment]
rabble rabble

**Pull request closed without merging**